NavigationReactNative/src/ios/NVNavigationBarView.m

```diff
@@ -19,6 +19,21 @@ - (void)didSetProps:(NSArray<NSString *> *)changedProps
     if ([changedProps containsObject:@"title"]) {
         [self.reactViewController.navigationItem setTitle:self.title];
     }
+    if ([changedProps containsObject:@"barTintColor"]) {
```

Comment:
We don't need to check if the props have changed. We can always set the colors even if they haven't changed. I needed to check for the `title` because the `title` can be set in two different places. If it isn't set by the `NavigationBar` then I don't want to overwrite any existing value. We don't have that problem with colors because they're only set by the `NavigationBar`.